### ISSUE TITLE
Changing the size of one tileset image shifts tiles in all tile layers

### ISSUE THREAD

**Issue by nene** [deepnight/ldtk#547]
For example, when I have a 128x96 px image of 16x16 tiles and I add one additional 16px column to it, so it becomes a 144x96 px image, the Ldtk correctly identifies that tileset image dimensions have changed and correctly recalculates tile layers that use this tileset. So in JSON the `px` and `src` coordinates remain the same but tile indexes (`t`) change:

```
  "__tilesetDefUid": 1,
  "gridTiles": [
-     { "px": [0,0], "src": [0,64], "f": 0, "t": 32, "d": [0] },
+     { "px": [0,0], "src": [0,64], "f": 0, "t": 36, "d": [0] },
```

This is all great. But in addition to layers that reference this tileset, all other tile layers also get recalculated. In JSON one can see that in addition to tile indexes (`t`) also the `src` coordinates change:

```
  "__tilesetDefUid": 32,
  "gridTiles": [
-     { "px": [192,48], "src": [0,32], "f": 0, "t": 12, "d": [99] },
+     { "px": [192,48], "src": [16,32], "f": 0, "t": 13, "d": [99] },
```

I believe this is a bug.

Currently my workaround is to manually look at the JSON diff and revert the changes in wrong tile layers, but it's quite a nuisance.

Besides this though, it's really an awesome level-editor. Thanks!

**Comment by nene** [comment]
Looking at the code I see that there's a [TilesetDef.remapAllTiles](https://github.com/deepnight/ldtk/blob/master/src/electron.renderer/data/def/TilesetDef.hx#L245) function, which seems to go through all layers regardless of which tileset they reference and remap them all.

Though I might be mistaken... it's my first time looking at the Ldtk source and Haxe code in general.

**Comment by deepnight** [comment]
Hi! Thanks for reporting. This bug should be fixed in 0.10 branch which should be released in January

Ref: #300

**Comment by nene** [comment]
Thanks. I'll see if I can manage to try the 0.10 branch out.
